poetry.lock

```diff
@@ -227,7 +227,7 @@ description = "Code coverage measurement for Python"
 name = "coverage"
 optional = false
 python-versions = ">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, <4"
-version = "5.0.2"
+version = "5.0.3"
```

Comment:
If I knew how, I would have updated poetry.lock just for the changes in pyproject.toml without updating unrelated packages.